### PR TITLE
add script bonita-start.sh in Tomcat bundle, to init the platform before starting

### PR DESCRIPTION
**Pull request by educhastenier**
This is preliminary step to [BS-15303](https://bonitasoft.atlassian.net/browse/BS-15303)

### CODE REVIEW

File: bundle/tomcat/pom.xml [hunk 1]
```diff
@@ -18,8 +18,6 @@
         <tomcat.home.dir>${tomcat.bos.home.dir}/apache-tomcat-${tomcat.version}</tomcat.home.dir>
         <bonita.home.dir>${tomcat.home.dir}/bonita</bonita.home.dir>
         <platform.setup.folder>${tomcat.home.dir}/setup</platform.setup.folder>
-        <bonita.home.server.default.conf.dir>${bonita.home.dir}/server/default/conf</bonita.home.server.default.conf.dir>
-        <bonita.home.client.dir>${bonita.home.dir}/client/</bonita.home.client.dir>
```
Comment (educhastenier) [inline bundle/tomcat/pom.xml]: Clean-up of useless properties

File: bundle/tomcat/src/main/resources/bonita-start.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Tests the return code of a command and terminate with error if different from 0.
+# usage: testReturnCode $? "an error message"
+# param 1: the return code
+# param 2: error message
+testReturnCode() {
```
Comment (laurentleseigneur) [inline bundle/tomcat/src/main/resources/bonita-start.sh]: should we keep this unused method or perform some db vendor check?

**Comment by laurentleseigneur** [comment]
LGTM, bonita-start.sh start bundle

File: bundle/tomcat/src/main/resources/bonita-start.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```
Comment (laurentleseigneur) [inline bundle/tomcat/src/main/resources/bonita-start.sh]: use standart shebang line

File: bundle/tomcat/assembly.xml [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
+        </fileSet>
+        <fileSet>
+            <outputDirectory>/</outputDirectory>
+            <directory>${tomcat.home.dir}</directory>
+            <includes>
+                <include>bin/*.sh</include>
+                <include>setup/*.sh</include>
+                <include>*.sh</include>
+            </includes>
+            <fileMode>0755</fileMode>
```
Comment (laurentleseigneur) [inline bundle/tomcat/assembly.xml]: fileMode used in platform-setup is 0740 for sh, after sam's feedback

File: deploy/distrib/pom.xml [hunk 5]
```diff
@@ -249,6 +249,11 @@
                                 <copy todir="${platform_setup_folder}">
                                     <fileset dir="${project.build.directory}/platform-setup" />
                                 </copy>
+                                <chmod perm="u+x">
```
Comment (laurentleseigneur) [inline deploy/distrib/pom.xml]: we don't need +r in perm ?

**Comment by laurentleseigneur** [comment]
linked with https://github.com/bonitasoft/bonita-distrib-sp/pull/28

**Comment by laurentleseigneur** [comment]
LGTM

**Pull request merged**